styles/tree-view.less

```diff
@@ -59,3 +59,7 @@
     }
   }
 }
+
+.name.dragenter {
+  color: #949292;
```

Comment:
Since this is a hard-coded color, it won't change to reflect the user's active UI theme. Would you mind changing this to [use one of the UI variables](https://atom.io/docs/latest/hacking-atom-creating-a-theme#variables)?